tas-installer/internal/uninstall/uninstall.go

```diff
@@ -0,0 +1,31 @@
+package uninstall
+
+import (
+	"fmt"
+
+	"securesign/sigstore-ocp/tas-installer/pkg/helm"
+	"securesign/sigstore-ocp/tas-installer/pkg/kubernetes"
+)
+
+func HandleHelmChartUninstall(tasNamespace, tasReleaseName string) error {
+	fmt.Println("Uninstalling helm chart")
+	info, err := helm.UninstallTrustedArtifactSigner(tasNamespace, tasReleaseName)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("Uninstalled helm release: %s namespace: %s %s\n", info.Release.Name, info.Release.Namespace, info.Info)
```

Comment:
See comment above re: user facing i/o in `pkg`. Might be worth considering moving it?

Reply:
will do

Reply:
done